parsers/discord.py

```diff
@@ -93,11 +93,9 @@ class Discord(ParserWindow):
     _bg_opacity = 25
 
     def __init__(self):
+        self.name = "discord"
```

Comment:
It occurs to me we probably don't need to set this in the init... just set it at class var level, it's essentially a static constant

Reply:
Yeah we can work on final layouts later on i was my thoughts. I already have more changes needed to this window as well after my other work :p

Reply:
The idea here is all the functionality still works as is currently and is testable. We can refactor the implementation a bit later on since things are going to shift a bit still as everything gets untangled.

Reply:
We also were previously setting the self.name the way it is done there, that behavior is the same as before.